test/parallel/test-fs-read-stream.js

```diff
@@ -35,7 +35,13 @@ const rangeFile = fixtures.path('x.txt');
   let paused = false;
   let bytesRead = 0;
 
-  const file = fs.createReadStream(fn);
+  const file = fs.createReadStream(fn, {
```

Comment:
rather than passing in a `fs` object, just pass in the `open`, `read`, and `close` directly on the `options` object... like so. With the change I suggest above, this would make the API a bit simpler.

```js
fs.createReadStream(fn, {
  open: common.mustCall(fs.open),
  read: common.mustCallAtLeast(fs.read, 1),
  close: common.mustCall(fs.close)});
```